.travis/linux.win64.install.sh

```diff
@@ -3,10 +3,12 @@ set -e
 
 # First, install 32-bit deps
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-. $DIR/linux.win32.install.sh
+# Maybe find a better fix: see https://github.com/koalaman/shellcheck/wiki/SC1090
+# shellcheck disable=SC1090
```

Comment:
Ok, so, after looking into this, seems to me it would be useless to do something else. Just disabling the sc is enough IMHO.